src/pages/code-of-conduct.js

```diff
@@ -80,11 +80,11 @@ export default function() {
       </PageSection>
       <PageSection
         title="Need help?"
-        description="Contact one of the organizars through twitter or personally."
+        description="Contact one of the organizers through Twitter or personally."
       >
         <div className="flex flex-wrap">
           {ORGANIZERS.map((organizer, index) => (
-            <Avatar key={index} {...organizer} />
+            <SocialAvatar key={index} {...organizer} />
```

Comment:
Suggestion: Using `index` as a key is a no-no. I would recommend using the `twitterLink` as the `key` because it is both unique and predictable. I guess that right now it doesn't matter because the layout will not change after the first render, but it could cause issues if at some point that list is either filterable or sortable.

Reply:
Makes sense, thanks for the suggestion!

Reply:
Good point, thanks! Although I guess right now there is no real consistent and guaranteed-unique attribute for organizers :(